README.md

```diff
@@ -51,12 +136,24 @@ pip install pytest
 pytest test
 ```
 
-TODO
------
+Migration Guide
+----------------
 
-Get the docs on read the docs
+The `setgapth.py` way to install is now deprecated by `pip install galgebra` and all modules in GAlgebra should be imported from `galgebra`, for example:
```

Comment:
Here I'd explicitly state what one would be migrating from; even though it's mentioned above, it's more friendly if sections of a document are relatively autonomous, not requiring reading the entire thing from star to finish.

Reply:
Yeah, I intended to write 2 subsections for users from `sympy.galgebra` and `brombo/galagebra` but only copy the latter migration guide and forgot the former. Will rectify that in a sec.

Reply:
Rectified